### PR TITLE
PE-10 styling

### PR DESCRIPTION
**Pull request by alirni**
PE-10 #done #comment tnx for merge

### CODE REVIEW

File: panel-filter.html [hunk 1]
```diff
@@ -6,19 +6,28 @@
...
+`--primary-color`|Mixin applied to the `:host`|`currentcolor`
+
 @demo demo/index.html
 -->
 
 <dom-module id="panel-filter">
 	<template>
 		<style>
 			:host {
+				--primary-color: var(--google-blue-500);
```
Comment (alimd) [inline panel-filter.html]: Please rename `--primary-color` to `--panel-filter-primary-color`

Comment (alimd) [reply panel-filter.html]: Please add `--panel-filter-text-color` with default value

Comment (alimd) [reply panel-filter.html]: Please add `--panel-filter` mixing

**Review by alimd — COMMENTED** [review]
Please add material theme in demo file and use material variable

**Comment by AliMD** [comment]
Please add material theme in demo file and use material variable

File: panel-filter.html [hunk 2]
```diff
@@ -27,9 +36,13 @@
 				box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 1px 4px 0 rgba(0, 0, 0, 0.12), 0 3px 1px -2px rgba(0, 0, 0, 0.2);
 			}
 
-			/*iron-icon {
-				padding-right: 5px;
-			}*/
+			.color {
+				color: var(--primary-color);
```
Comment (alimd) [inline panel-filter.html]: Use ptc-style var colors

File: panel-filter.html [hunk 3]
```diff
@@ -54,7 +67,21 @@
...
+
 			}
+
+			.iron-icon-green {
+				--iron-icon : {
+					color: var(--google-green-500);
+				};
+			}
+
+			.iron-icon-red {
```
Comment (alimd) [inline panel-filter.html]: follow our new classname guideline

**Review by alimd — CHANGES_REQUESTED** [review]

**Comment by alimd** [comment]
Use PE-18 for styling

File: panel-filter.html [hunk 4]
```diff
@@ -132,6 +139,11 @@
 					type : Number
 				},
 
+				elevation: {
```
Comment (alimd) [inline panel-filter.html:142]: 👍

**Review by alimd — APPROVED** [review]
perfect

**Pull request merged**